doc-content/enterprise-only/admin-and-config/exporting-importing-dashbuilder-data-proc.adoc

```diff
@@ -150,3 +150,49 @@ WARNING: You should only import Dashbuilder data to a clean installation of {PRO
 . Click *Import*.
 
 endif::DM[]
+
+ifdef::PAM[]
+
+== Deploying dashboards from {CENTRAL} on Dashbuilder Runtime
+
+You can automatically deploy the dashboards from {CENTRAl} on Dashbuilder Runtime. {CENTRAl} is linked to Dashbuilder Runtime with the help of gradual export feature.
+
+.Prerequisites
+
+* Dasbuilder runtime is configured on the system.
+* You have set the `dashbuilder.runtime.multi` system property to `true` in the `standalone.xml` file.
+* You have set the value of `dashbuilder.runtime.location` system property to the dashboard root URL.
++
+For example:
++
+[source]
+----
+<property name="dashbuilder.runtime.location" value=" http://localhost:8080"
+----
+* You have set the `dashbuilder.export.dir` system property to the shared directory where Dashbuilder Runtime reads its model.
++
+For example:
++
+[source]
+----
+<property name="dashbuilder.export.dir" value="/tmp/dashbuilder/models/"
+----
+
+
+.Procedure
+
+. Start {CENTRAL}, go to *Menu → Design → Pages*.
+. In the *Components* panel, drag and edit the required component types to the canvas.
+. Click *Save*.
+. In {CENTRAL}, select the *Admin* icon in the top-right corner of the screen and select *Dashbuilder Data Transfer*.
+. On the *Dashbuilder Data Transfer* page, click *Custom export*.
+. In *Export Wizard* panel, select the pages that you want to include in the ZIP file and click *Next*.
+. Select *Export Wizard* and click *Open*.
++
+You can see the Dashbuilder Runtime home page. If you are not logged in, you are redirected to the log in page.
```

Comment:
I think it's "login" page, not two words

Reply:
Thank you for noticing this typo. I have updated in the content.